names/service_account.go

```diff
@@ -81,7 +81,7 @@ func EnsureValidServiceAccountId(id string) (string, error) {
 	}
 
 	if !IsValidServiceAccountId(id) {
-		return "", errors.E(errors.CodeBadRequest, "invalid client ID")
+		return "", errors.New("invalid client ID")
```

Comment:
could we make 
```
var InvalidClientIDError = errors.New("invalid client ID")
```
so that users of this method can check for this specific error?